main.go

```diff
@@ -0,0 +1,150 @@
+package main
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"sort"
+)
+
+type Entry struct {
+	ID          uint32
+	FirstName   string
+	LastName    string
+	PhoneNumber string
+}
+
+func check(err error) {
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+func Remove() string {
+	const phonebookFile = "phonebook.json"
+	var entries []Entry
+	f, err := os.Open(phonebookFile)
+	check(err)
+	defer f.Close()
+	if err := json.NewDecoder(bufio.NewReader(f)).Decode(&entries); err != nil {
+		log.Fatal(err)
+	}
+	id := -1
+	fmt.Println("Enter ID:")
+	fmt.Scanln(&id)
+	for i, v := range entries {
+		if v.ID == uint32(id) {
+			entries = append(entries[:i], entries[i+1:]...)
+			f, _ = os.Create("phonebook.json")
```

Comment:
too nested. would be nice to make it flat and factor out things into separate functions e.g.
```
func Remove() ... {
  entries := LoadPhonebook()
  var id uint32
  fmt.Print("Enter ID to remove: ")
  fmt.Scan(&id)
  entries = removeByID(entries, id)
  savePhonebook(entries)
}
```